etna/search/views.py

```diff
@@ -583,7 +583,7 @@ class CatalogueSearchView(BucketsMixin, BaseFilteredSearchView):
     default_group = "tna"
     form_class = CatalogueSearchForm
     template_name = "search/catalogue_search.html"
-    title_base = SearchTabs.CATALOGUE.value
```

Comment:
I can see the class attribute name connected to the class name , before both had base on them

Reply:
Update to use meaningful name. Its the search tab eventually.